<launch>
    <node pkg="reference_path_generator" type="reference_path_generator" name="reference_path_generator" output="screen">
        <param name="frame_id"  type="string"   value="odom"/>
        <!--

        -->
        <!-- Radius <= 5.0; k <= 1.0 -->
        <param name="x_radius"  type="double"   value="5.0"/>
        <param name="y_radius"  type="double"   value="5.0"/>

        <!-- x,y_func can be sin, cos, default = sin -->
        <param name="x_func"    type="string"   value="sin"/>   
        <param name="y_func"    type="string"   value="cos"/>
        
    </node>
</launch>